### ISSUE TITLE
Lint/DisjunctiveAssignmentInConstructor cop can't handle empty constructors

### ISSUE THREAD

**Issue by ashmaroli** [rubocop/rubocop#6677]
`Lint/DisjunctiveAssignmentInConstructor` cop throws an error when inspecting an empty constructor.

## Expected behavior

Either flag an offense stating that empty constructors are not recommended or ignore and proceed with inspection.

## Actual behavior

```
An error occurred while Lint/DisjunctiveAssignmentInConstructor cop was inspecting [ruby_file.rb]
```

## Sample to reproduce the problem

```ruby
class Foo
  def initialize(options)
    # no op.
    # override this method in subclass.
  end

  [...]
end
```
## RuboCop version

```
$ [bundle exec] rubocop -V
0.63.0
```

**Comment by rmm5t** [comment]
I have a PR that fixes this here: https://github.com/rubocop-hq/rubocop/pull/6679

**Comment by ashmaroli** [comment]
Technically, #6678 is the duplicate because it was opened *after* this ticket.. :wink: :wink:
